### DISCUSSION TITLE
关于文档

### DISCUSSION THREAD

**Discussion by zuihou** [Snailclimb/JavaGuide#1975] [General]
![image](https://user-images.githubusercontent.com/8500410/231982408-51bfb2db-ceb3-4f2b-94bb-fd89bf276e15.png)

最新在学习使用 vuepress-theme-hope，请问您左侧sidebar是如何实现

1. 固定在左侧
2. 搜索按钮如何调出来

**Comment by Snailclimb** [comment]
> ![image](https://user-images.githubusercontent.com/8500410/231982408-51bfb2db-ceb3-4f2b-94bb-fd89bf276e15.png)
> 
> 最新在学习使用 vuepress-theme-hope，请问您左侧sidebar是如何实现
> 
> 1. 固定在左侧
> 2. 搜索按钮如何调出来

这个是比较新的  vuepress-theme-hope 版本才有的。

**Reply by zuihou** [reply]
恩，我用的最新版本， 改那个参数配置成这样呢？ 他默认是这样的：


![image](https://user-images.githubusercontent.com/8500410/232050899-1a7c2839-3ce1-4d04-9554-d5f7745a8042.png)

**Reply by Snailclimb** [reply]
> 恩，我用的最新版本， 改那个参数配置成这样呢？ 他默认是这样的：
> 
> ![image](https://user-images.githubusercontent.com/8500410/232050899-1a7c2839-3ce1-4d04-9554-d5f7745a8042.png)

我忘了，你去看看官方文档就好。
